### PR TITLE
Update NearFutureSolar-Core for 1.1, primary moved to SpaceDock

### PR DESCRIPTION
**Pull request by lOmicronl**

### CODE REVIEW

**Comment by lOmicronl** [comment]
Not sure why the build check failed. JSONLint says the syntax is valid...

**Comment by plague006** [comment]
You can't use `find` on a file without including `"find_matches_files": true` in the install stanza.

File: NetKAN/NearFutureSolar-Core.netkan [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "spec_version"        : "v1.4",
+    "identifier"          : "NearFutureSolar-Core",
+    "$kref"               : "#/ckan/spacedock/559",
+    "name"                : "Near Future Solar Core",
+    "abstract"            : "This is the Near Future Solar plugin stand-alone, for adding functionality to other mods. It contains no parts and does nothing by itself.",
+    "author"              : "Nertea",
+    "license"             : "CC-BY-SA-NC-4.0",
```
Comment (plague006) [inline NetKAN/NearFutureSolar-Core.netkan]: `"CC-BY-NC-SA-4.0"`

**Comment by lOmicronl** [comment]
Thanks, I think I fixed it by changing the "find" to a "file"... yep, the build suceeds now.

In regards to the license - how odd! I thought it was the wrong way around, because SpaceDock displays it differently on the mod pages, so I went and changed it everywhere. Guess CKAN didn't like that! ;)

**Comment by plague006** [comment]
Looks good, merging with my thanks.

**Pull request merged**